### PR TITLE
[Sage-515] Sage 3 - Panel Updates

### LINKED ISSUE #515: Update Panel
## Description

Update component and subcomponents to match latest specs in Sage 3.0. The following were noted during initial audit:

- Double-check border-radius
- Add subtext block
- Consider whether header changes should also be synced in Card context.

**Dependencies**:
- [Border radius](https://github.com/Kajabi/sage-lib/issues/472)

**Links**
- Figma specs:
	- [Panel](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/WIP-Sage-3-%E2%80%94-Admin-Components?node-id=3561%3A9796)
	- [Headings](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/WIP-Sage-3-%E2%80%94-Admin-Components?node-id=3909%3A1134)
- [Existing component(s)](https://sage-design-system.kajabi.com/pages/component/panel)

### PR DESCRIPTION
**Pull request by FuturaExtraBold**
## Description
Updates the Panel Rails and React Component styles and markup per #515. Double-checks the border-radius is correct. A new subtext block has been added to the Panel header and subheader, that accepts arbitrary markup.

## Screenshots

|  Before  |  After  |
|--------|--------|
|image|image|

### Rails

- View [Rails component](http://localhost:4000/pages/component/panel)
- Check that component UI exists and nothing is broken
- Ensure the subtext content can be added

### React
- View [Storybook](http://localhost:4100/?path=/docs/sage-panel--default)
- Check that component UI exists and nothing is broken
- Ensure the subtext content can be added

## Testing in `kajabi-products`
(MEDIUM) Alters styling and adds markup for all Panels in Kajabi Products. Please sanity-test that Panels render correctly, including this new subtext block.

## Related
Closes #515 
Reverted from #752 
Needs to be released with https://github.com/Kajabi/kajabi-products/pull/20552

### CODE REVIEW

**Comment by FuturaExtraBold** [comment]
Please review so this and https://github.com/Kajabi/kajabi-products/pull/20552 can be ready to merge in tandem for the next bump.

**Review by anechol — APPROVED** [review]
🚀

File: docs/app/views/examples/components/panel/_preview.html.erb [hunk 1]
```diff
@@ -17,7 +17,10 @@ Ut elementum id leo ac fringilla.
 <%= sage_component SagePanelHeader, { title: "Basic Panel" } %>
 
 <%= sage_component SagePanel, {} do %>
-  <%= sage_component SagePanelHeader, { title: "Panel Header" } do %>
+  <%= sage_component SagePanelHeader, {
+    # subtext: "<p>All kinds of awesome, wonderful content can go in here!</p>".html_safe,
```
Comment (pixelflips) [inline docs/app/views/examples/components/panel/_preview.html.erb:21]: Is it safe to remove the commented-out line?

**Review by pixelflips — APPROVED** [review]
One tiny nitpick but LGTM!

**Review by philschanely — APPROVED** [review]

File: docs/app/views/examples/components/panel/_preview.html.erb (hunk 1 above)
Comment (FuturaExtraBold) [reply docs/app/views/examples/components/panel/_preview.html.erb]: Removed. Good catch.

**Pull request merged**